backend/blog/views.py

```diff
@@ -25,3 +33,45 @@ def blog_post(request, slug):
     tags = Tag.objects.all()
     context = {'post': post, 'tags': tags}
     return render(request, 'blog/post.html', context)
+
+
+def user_register(request):
+    if request.method == "POST":
```

Comment:
Hey – we should talk about this. This actually opens us up to anyone being able to register and therefore add/delete/edit blog posts. This is very dangerous! I appreciate all the effort that you put in this PR. 
Like we talked with @ADJohnson314 I think Django does a really great job with the bare bones registration that exists already. I was expecting only some permissions settings, which you're doing on any new user registration, but you're also doing so much more! Great exploration but I think this means we need to rethink our set up.